service_event/src/test/scala/DateOrderingTest.scala

```diff
@@ -0,0 +1,55 @@
+/**
+ * Created by jarle on 23.08.16.
+ */
+import java.sql.{Date, Timestamp}
+
+import no.uio.musit.microservice.event.service.EventOrderingUtils
+import no.uio.musit.microservice.event.service.EventOrderingUtils._
+import org.joda.time.DateTime
+import org.scalatestplus.play.PlaySpec
+class DateOrderingTest extends PlaySpec {
+
+  "testDateOrdering" must {
+    "some date tests must succeed" in {
+
+      val today = new java.sql.Date(DateTime.now().getMillis)
+      val yesterday = new java.sql.Date(DateTime.now().getMillis - 24 * 60 * 60 * 1000)
+
+      val optYesterday = Some(yesterday)
+      val optToday = Some(today)
+
+
+      val ordering = EventOrderingUtils.optionOrdering(sqlDateOrdering)
+      ordering.compare(optYesterday, optToday) mustBe -1
+      ordering.compare(optYesterday, optToday) mustBe -1
+      ordering.compare(optToday, optYesterday) mustBe 1
+      ordering.compare(None, None) mustBe 0
+      ordering.compare(None, optYesterday) mustBe -1
+
+      val ordering2 = EventOrderingUtils.optionOrdering[Date]
```

Comment:
Ordering of java.util.Date and java.sql.Date should be separated into 2 different test cases.
As a general rule, one test case should test one thing. In this case testing the comparison of java.util.Date and java.sql.Date are two test cases.